src/shared/components/plots/TablePlot.tsx

```diff
@@ -135,9 +135,7 @@ export default class TablePlot extends React.Component<ITablePlotProps, {}> {
 
     @computed get tableData(): ITableData {
         // count by categories
-        const tableCounts: StringListIndexedMap<
-            number
-        > = new StringListIndexedMap();
```

Comment:
[Keep type parameters inline for type annotations in variable declarations](https://prettier.io/blog/2019/11/09/1.19.0.html#keep-type-parameters-inline-for-type-annotations-in-variable-declarations-6467-by-sosukesuzuki)